### PR TITLE
ENT-3451: Make php version detection more dynamic

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: cfe_internal/enterprise/CFE_hub_specific.cf [hunk 1]
```diff
@@ -48,6 +48,20 @@ bundle common cfe_internal_hub_vars
       "SSLCertificateKeySize" string => "4096";
       "SSLCertificateDaysValid" string => "3650";
 
+      # Determine the version of PHP that is used
+
+      "php_version"
+        string => ifelse( fileexists( "$(sys.workdir)/httpd/modules/libphp7.so" ), "7",
+                          fileexists( "$(sys.workdir)/httpd/modules/libphp5.so" ), "5"
```
Comment (craigcomstock) [inline cfe_internal/enterprise/CFE_hub_specific.cf]: need a comma here I think, after "5" and before "UNKNOWN"

Comment (nickanderson) [reply cfe_internal/enterprise/CFE_hub_specific.cf]: Yes indeed
good catch.

**Comment by jimis** [comment]
is this one ready for merging

**Comment by nickanderson** [comment]
@jimis approved, merging.

**Pull request merged**